script/foreman-puma-status

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+scl enable tfm 'puma-status /tmp/puma/pids/puma.state'
```

Comment:
We've always kept SCL out of the main Foreman repository because it's only for EL7. It doesn't make sense in Debian, Ubuntu and EL8. This is something we apply in packaging.

I also wonder how this path is chosen. Foreman has a temp dir and perhaps that makes more sense? Or `/run/foreman/puma.state`

Reply:
> We've always kept SCL out of the main Foreman repository because it's only for EL7. It doesn't make sense in Debian, Ubuntu and EL8. This is something we apply in packaging.

For my understanding, if I just keep `puma-status /tmp/puma/pids/puma.state`, how does it work?

Reply:
> I also wonder how this path is chosen. Foreman has a temp dir and perhaps that makes more sense? Or `/run/foreman/puma.state`
 
I missed pushing a change yesterday, I intended to use `/usr/share/foreman/tmp/pids/` since we already have it created for other process ids.  I am reusing this folder now, let me know your thoughts.